src/server_manager/install_scripts/install_server.sh

```diff
@@ -406,5 +433,54 @@ ${FIREWALL_STATUS}
 END_OF_SERVER_OUTPUT
 } # end of install_shadowbox
 
-# Wrapped in a function for some protection against half-downloads.
-install_shadowbox
+function read_param() {
+  if (( "$#" < 2 )); then
+    log_error "Missing $1 value"
+    display_usage
+    exit 1
+  fi
+  echo $2
+}
+
+function parse_flags() {
+  local PARAMS=""
+  while [[ "$#" > 0 && ! "$1" == "--" ]]; do
```

Comment:
Should we use `getopts`? We wouldn't need to call eval, which could open a vulnerability.